consult-yasnippet.el

```diff
@@ -0,0 +1,156 @@
+;;; consult-yasnippet.el --- Provides the command `consult-yasnippet' -*- lexical-binding: t; -*-
+
+;; Author: Mohsin Kaleem, Consult and Selectrum contributors
```

Comment:
Here the original authors should be acknowledged.

Reply:
Oooh, I wasn't sure what to put there. Thanks for letting me now. :smiley:.

Reply:
I think it is fair to write `<name of original authors>, Mohsim Kaleem, Daniel Mendler`. Then we should also add a comment that it is derived from ivy-yasnippet and a link to ivy-yasnippet in the commentary section. Maybe you already have that. I haven't looked.

Reply:
I don't really care if my names in their but I agree we should say some of it was derived from `counsel-yasnippet`. Looks like the venerable alphapapa advises putting a [credits](https://github.com/alphapapa/emacs-package-dev-handbook#template) in the package docstring to reference helpful contributors.

I've added that now. How does this sound?

```lisp
;; This package would not have been possible without the ivy-yasnippet[1]
;; package which showed me how to access and preview yasnippet templates
;; through a `completing-read' interface.
;;
;;  [1] https://github.com/mkcms/ivy-yasnippet
```

Reply:
I think it is better to write something like "This package has been derived from ... by ..." and some link. It is not as if we just looked up some tricks from there. It is clearly a derivative.

Reply:
Plain and simple?

```lisp
;; This package has been derived from the ivy-yasnippet[1] package.
;;
;;  [1] https://github.com/mkcms/ivy-yasnippet
```

Reply:
Yes something like this but the author names should also go there. It is not okay if we add our names but omit the original authors ;) But look, these things are not crucial. I will fix this afterwards if necessary. I don't want to over discuss things.

Reply:
I added it at the top just before your name like you sugggested before... would you like me to add it in both places?